src/quo2/components/tags/network_tags/view.cljs

```diff
@@ -0,0 +1,83 @@
+(ns quo2.components.tags.network-tags.view
+  (:require [quo2.components.tags.base-tag :as base-tag]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.tags.network-tags.style :as style]
+            [react-native.core :as rn]))
+
+(defn outer-resource-container
+  [size background-color]
+  {:background-color background-color
+   :border-radius    size
+   :width            size
+   :height           size
+   :margin-left      (case size
+                       32 -12
+                       24 -8)
+   :align-items      :center
+   :justify-content  :center})
+
+(defn network-group
+  []
+  (fn [{:keys [network-icon size background-color]
+        :or   {size 24}}]
+    [rn/view
+     {:style (style/network-tags-child)}
+     ^{:key network-icon}
+     [rn/view {:flex-direction :row}
+      [rn/view (outer-resource-container size background-color)
+       [rn/image
+        {:source network-icon
+         :style  {:height        (case size
+                                   32 28
+                                   24 20)
+                  :width         (case size
+                                   32 28
+                                   24 20)
+                  :border-radius size}}]]]
+    ]))
+
+(defn selected-group-count
+  [networks]
+  (cond
+    (> (count networks) 3) 3
+    :else
+    (count networks)))
+
+(defn tag-networks
+  []
+  (fn [{:keys [networks size background-color]
+        :or   {size 24}}]
+    (let [selected-groups (take (selected-group-count networks) networks)]
+      [rn/view
+       {:style (style/network-tags-child)}
+       (for [{:keys [network-icon id]} selected-groups]
+         ^{:key id}
+         [network-group
+          {:network-icon     network-icon
+           :size             size
+           :background-color background-color}])])))
+
+(defn network-tags
+  [_ _]
+  (fn [{:keys [title networks size background-color on-press]
+        :or   {size 24}}]
+    [rn/view
+     {:style (style/network-tags-container size)}
+     [base-tag/base-tag
+      {:size     size
+       :type     :permission
+       :on-press on-press}
+      [rn/view
+       {:style (style/network-tag-view)}
+       [tag-networks
+        {:networks         networks
+         :size             size
+         :background-color background-color}]
+       [rn/view
+        {:padding-left (case 32
+                         8 24
+                         6)}
+        [rn/text
```

Comment:
here you should be using "quo2" `text` component ->
you import/require it from here: "quo2.components.markdown.text/text"

color looks wrong there (🤔) but if you use this component you will not need to set color anyway 👍 
```
[:quo2.components.markdown.text :as text]
...
[text/text {:size :paragraph-2
:weight :medium
} title]

```